src/02-common-principles.md

```diff
@@ -383,6 +383,200 @@ In particular, if a BIDS dataset contains a `derivatives/` sub-directory,
+
+```json
+{
+    "DatasetLinks": {
+        "deriv2": "https://mydatahost.com/derivatives/derivative2"
+    }
+}
+```
+
+This means that the needed files can be obtained by downloading the data
+from `https://mydatahost.com/derivatives/derivative2`, and then navigating
+to `/sub-01/anat/sub-01_desc-preproc_T1w.nii.gz` relative to that
+downloaded data.
+
+However,
+arbitrary remote server locations such as the `mydatahost.com` example server may not be stable.
+Therefore it is RECOMMENDED to instead point to archived versions of a datasets using their DOIs:
+
+```json
+{
+    "DatasetLinks": {
+        "deriv3": "doi:10.18112/openneuro.ds003669.v1.0.0"
+    }
+}
+```
+
+Again, this means that the linked dataset must be downloaded from the DOI address,
+and the file will be found by navigating to the location specified in the BIDS URI
+relative to the downloaded data.
+
+### Refer to a file outside of a dataset but on the same host
+
+It is RECOMMENDED to refer to files **within** a dataset,
+or on a **remote** location as described in the sections above.
+However sometimes it may be convenient to refer to files that are outside of a given dataset but on the same host.
+BIDS URIs allow for specifying such locations,
+but such specifications are by definition not portable in that the BIDS URIs break when the host changes.
+When sharing a BIDS dataset, the dataset curator MUST make sure that all BIDS URIs are portable.
+
+To link for example to a derivative file that is stored in a `derivatives/`
+directory that is NOT nested in the raw BIDS directory:
+
+`bids:deriv3:/sub-01/anat/sub-01_desc-preproc_T1w.nii.gz`
+
+```json
+{
+    "DatasetLinks": {
+        "deriv3": "file://../some-local-path/derivative3"
+    }
+}
+```
+
+As shown in the example above, the path to `deriv3` contains the `..` syntax,
+which is a common way to refer to parent directories.
+In this way, arbitrary locations on the host can be referenced relative to the current dataset.
+
+Alternatively, one may specify an absolute location on the host using a
+forward slash directly after the `file://` URI scheme as such:
+`file:///`.
+
+On Unix-like operating systems (including MacOS) this refers to the system root.
+One may reference a dataset on some example user's Desktop as such:
+
+```json
+{
+    "DatasetLinks": {
+        "deriv3": "file:///home/example-user/Desktop/my-dataset"
+    }
+}
+```
+
+On the Windows operating system, `file:///` (note the `/` after the `file://` URI scheme)
+refers to a "virtual" system root that does not exist in practice.
+To reference a dataset on our example user's Windows Desktop, one may use:
+
+```json
+{
+    "DatasetLinks": {
+        "deriv3": "file:///C:/Users/example-user/Desktop/my-dataset"
+    }
+}
+```
+
+On a Windows machine, the location from this example would look like:
+`C:\Users\example-user\Desktop\my-dataset`
+
+However throughout BIDS, forward slashes `/` MUST be used as the path separator.
```

Comment:
it would be good to clarify this requirement also in the [common principles](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html)

Reply:
I agree, but this is orthogonal to the present PR so I opened a new issue for this: https://github.com/bids-standard/bids-specification/issues/849